packages/react-scripts/template/README.md

```diff
@@ -885,19 +885,25 @@ Open your `package.json` and add a `homepage` field:
 **The above step is important!**<br>
 Create React App uses the `homepage` field to determine the root URL in the built HTML file.
 
-Now, whenever you run `npm run build`, you will see a cheat sheet with a sequence of commands to deploy to GitHub pages:
+Now, whenever you run `npm run build`, you will see a cheat sheet with instructions on how to deploy to GitHub pages:
 
 ```sh
-git commit -am "Save local changes"
-git checkout -B gh-pages
-git add -f build
-git commit -am "Rebuild website"
-git filter-branch -f --prune-empty --subdirectory-filter build
-git push -f origin gh-pages
-git checkout -
-```
+To publish it at http://myname.github.io/myapp, run:
+
+  npm install --save-dev gh-pages
 
-You may copy and paste them, or put them into a custom shell script. You may also customize them for another hosting provider.
+Add the following script in your package.json.
+
+    // ...
+    "scripts": {
+      // ...
+      "deploy": "gh-pages -d build"
+    }
+
+Then run:
+
+  npm run deploy
+```
```

Comment:
Can we only put the script and deployment command inside a code block? Now it looks a bit weird.

I mean like this (also changed the URL to http://myusername.github.io/my-app here): 

>To publish it at http://myusername.github.io/my-app, run:
>
>```
>npm install --save-dev gh-pages
>```
>
>Add the following script in your package.json:
>
>```
>  // ...
>  "scripts": {
>    // ...
>    "deploy": "gh-pages -d build"
>  }
>```
>
>Then run:
>
>```
>npm run deploy
>```